docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -167,17 +167,23 @@ The operation was successful if the returned status says `true`.
   Kyma Dashboard
   </summary>
 
-1. Using the left navigation, go back to **Workloads** > **Functions**.
-2. Select your `lastorder` Function and navigate to the **Configuration** tab.
-3. Click on **Add Event Subscription+**.
-4. Provide the following parameters:
-    - **Application name**: `myapp`
+1. Using the left navigation go to **Apllications**.
+2. Click on **Create Application+**.
+3. Provide a name for your application, for example `myapp`.
+4. Go to **Namespaces** and click on `default`.
+5. Using the left navigation, go back to **Workloads** > **Functions**.
+6. Select your `lastorder` Function and navigate to the **Configuration** tab.
+7. Click on **Create Subscription+**.
+8. Provide the following parameters:
+    - **Subscription name**, for example `received-orders`
```

Comment:
```suggestion
    - **Subscription name**: `lastorder-sub`
```
Note, that we also have steps for kubectl, so we should aim to keep them (including the values used) as consistent as possible, just in case someone prefers to perform some steps via kubectl and others via Kyma Dashboard.